Apps/WebClient/test/functional/e2e/cypress/integration/dependent/dependents.js

```diff
@@ -0,0 +1,120 @@
+            .blur()
+            .should('have.class', 'is-invalid')
+        // Mandad this is broken and needs to be fixed to validate
+        //   cy.get('[data-testid=dateOfBirthInput]')
+        //     .should('be.enabled')
+        // Validate PHN input
+        cy.get('[data-testid=phnInput]')
+            .should('be.enabled')
+            .clear()
+            .blur()
+            .should('have.class', 'is-invalid')
+        // Validate Gender Input  
+        cy.get('[data-testid=genderInput]')
+            .should('be.enabled')
+            .select('').contains('Please select an option')
+        cy.get('[data-testid=genderInput]')
+            .select(gendeMale).should('have.value', gendeMale)
+        cy.get('[data-testid=genderInput]')
+            .select(gendeFemale).should('have.value', gendeFemale)
+        cy.get('[data-testid=genderInput]')
+            .select(gendeUnknown).should('have.value', 'NotSpecified')
+        // Validate Cancel out of the form
+        cy.get('[data-testid=cancelRegistrationBtn]')
+            .should('be.enabled', 'be.visible')
+            .click()
+        // Validate the modal is done, again should use modal id    
+        cy.get('[data-testid=newDependentModalText]').should('not.exist')
+    })
+
+    it('Validate Add', () => {
+        cy.get('[data-testid=addNewDependentBtn]')
+            .click();
+        //Validate Modal appears - note this should use a modal ID but we don't have one
+        cy.get('[data-testid=newDependentModalText]').should('exist', 'be.visible')      
+        cy.get('[data-testid=firstNameInput]')
+            .type(firstName);
+        cy.get('[data-testid=lastNameInput]')
+            .type(lastName);
+        cy.get('[data-testid=dateOfBirthInput]')
+            .type(doB);
+        cy.get('[data-testid=phnInput]')
+            .type(phn);
+        const genderInput = cy.get('[data-testid=genderInput]')
+        genderInput.select(gendeFemale);
+        cy.get('[data-testid=termsCheckbox]')
+            .click({ force: true });
+
+        cy.get('[data-testid=registerDependentBtn]').click(); 
+
+        // Validate the modal is done, again should use modal id    
+        cy.get('[data-testid=newDependentModalText]').should('not.exist')
+
+        // You should now validate that the Tabs appear 
+    });
+
+    it('Validate Dependent Tab', () => {
+        // Validate the newly added dependent tab and elements are present        
+        cy.get('[data-testid=dependentPHN]')
+            .last().invoke('val')
+            .then(phnNumber => expect(phnNumber).to.equal(maskedPHN));
+        cy.get('[data-testid=dependentDOB]')
+            .last().invoke('val')
+            .then(dateOfBirth => expect(dateOfBirth).to.equal(doB));
+        cy.get('[data-testid=dependentGender]')
+            .last().invoke('val')
+            .then(dateOfBirth => expect(dateOfBirth).to.equal(gendeFemale));        
+    })
+
+    it('Validate Covid Tab', () => {
```

Comment:
We should probably comment this out as it currently fails and will be tested on a different test

Reply:
@mk08sh currently we can test if we can click on it and it shows "No Records" per screenshot above. I think we can add more tests to it once the covid issue is resolved

Reply:
per our call, the tests are working fine now.